stories/multi-select.stories.js

```diff
@@ -120,4 +121,31 @@ storiesOf("React Multi Select", module)
         showSelectAll={boolean("Show select all", true)}
       />
     );
-  });
+  })
+    .add("display custom selected item", () => {
+        const items = Array.apply(null, {length: 10}).map((i, index) => ({
+            id: index,
+            value: `item ${index}`,
+            label: <div><Icon>star</Icon>{`Item ${index}`}</div>
+        }));
+
+        const displaySelectedItemFunc = (item) => <div>{`selected: ${item.value}`}</div>;
+
+        const searchRenderer = ({searchTerm, onSearchTermChange}) => {
+            return (
+                <div>custom search <input value={searchTerm} onChange={onSearchTermChange}/></div>
```

Comment:
This story looks bad.
I think the wrapper component in our component (not searchRenderer) should have some sort of basic style to ensure that the component looks usable...

Reply:
@talyak the custom style storybook is now broken...